src/test/java/uk/ac/cam/cl/dtg/isaac/dao/EventBookingPersistenceManagerTest.java

```diff
@@ -0,0 +1,138 @@
+package uk.ac.cam.cl.dtg.isaac.dao;
+
+import static java.time.Instant.now;
+import static org.easymock.EasyMock.createMock;
+import static org.easymock.EasyMock.expect;
+import static org.easymock.EasyMock.replay;
+import static org.easymock.EasyMock.verify;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static uk.ac.cam.cl.dtg.CustomAssertions.assertDeepEquals;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Timestamp;
+import java.time.Instant;
+import java.util.List;
+import java.util.Map;
+import org.jetbrains.annotations.NotNull;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import uk.ac.cam.cl.dtg.isaac.dos.eventbookings.BookingStatus;
+import uk.ac.cam.cl.dtg.isaac.dto.IsaacEventPageDTO;
+import uk.ac.cam.cl.dtg.isaac.dto.eventbookings.DetailedEventBookingDTO;
+import uk.ac.cam.cl.dtg.isaac.dto.users.RegisteredUserDTO;
+import uk.ac.cam.cl.dtg.isaac.dto.users.UserSummaryWithEmailAddressAndGenderDTO;
+import uk.ac.cam.cl.dtg.segue.api.managers.UserAccountManager;
+import uk.ac.cam.cl.dtg.segue.auth.exceptions.NoUserException;
+import uk.ac.cam.cl.dtg.segue.dao.SegueDatabaseException;
+import uk.ac.cam.cl.dtg.segue.dao.content.ContentManagerException;
+import uk.ac.cam.cl.dtg.segue.dao.content.GitContentManager;
+import uk.ac.cam.cl.dtg.segue.database.PostgresSqlDb;
+
+class EventBookingPersistenceManagerTest {
+
+  private PostgresSqlDb mockEventsDatabase;
+  private UserAccountManager mockAccountManager;
+  private GitContentManager mockContentManager;
+  private ObjectMapper mockObjectMapper;
+  private EventBookingPersistenceManager eventBookingPersistenceManager;
+
+  @BeforeEach
+  public void setUp() {
+    mockEventsDatabase = createMock(PostgresSqlDb.class);
+    mockAccountManager = createMock(UserAccountManager.class);
+    mockContentManager = createMock(GitContentManager.class);
+    mockObjectMapper = createMock(ObjectMapper.class);
+    eventBookingPersistenceManager = new EventBookingPersistenceManager(mockEventsDatabase, mockAccountManager, mockContentManager, mockObjectMapper);
+  }
+
+  @Test
+  void testAdminGetBookingsByEventIds()
+      throws SegueDatabaseException, ContentManagerException, SQLException, NoUserException {
+    // Given
+    List<String> eventIds = List.of("event1", "event2");
+    String expectedQuery = "SELECT event_bookings.* FROM event_bookings JOIN users ON users.id=user_id WHERE event_id IN (?, ?) AND NOT users.deleted";
+    Instant now = now();
+
+    IsaacEventPageDTO eventPage1 = new IsaacEventPageDTO();
```

Comment:
Small one, but I think that all these could be mocks.

Reply:
Sorry, I should have explained this better. The reasoning was that if those DTOs do not affect the test at all (they just need to be passed because they're requirements of the function, but they're not interacted with), then making them a mock will help convey that. However, if we also need to start mocking the getters/setters of those DTOs, then in that case it's not worth it.